### ISSUE TITLE
Getting a SHA mismatch on new update

### ISSUE THREAD

**Issue by lor1x** [khanhas/spicetify-cli#1127]
## ℹ Computer information

- Spotify version: 1.1.69
- Spicetify version: 2.7.0

## 📝 Provide detailed reproduction steps (if any)

1. brew upgrade / brew install khanhas/tap/spicetify-cli

### ✔️ Expected result

spicetify-cli updates normally without issues

### ❌ Actual result

==> Downloading https://github.com/khanhas/spicetify-cli/archive/v2.7.0.tar.gz
Already downloaded: /Users/lori/Library/Caches/Homebrew/downloads/c1fba829db042aeb3b3dcc52c83d81bf7165c0764f3f7194d1532f23cd955e0f--spicetify-cli-2.7.0.tar.gz
Error: spicetify-cli: SHA256 mismatch
Expected: 9fb1e3bd97017a6638292bedc327c180765e9190fce80ebffffb2f7391ae33ff
  Actual: 3915c1d1667f66ad3a4c8cb6e27ee653da54351c431dac684b77d61b30dbfbcf

**Comment by lor1x** [comment]
Could @afonsojramos or @itsmeow take a look at this if you are free?

**Comment by afonsojramos** [comment]
@lor1x this is on me. I'll investigate, apologies for the inconvenience.

**Comment by lor1x** [comment]
Thank you!

**Comment by afonsojramos** [comment]
First of all, my thought process was checking the checksum of what I uploaded and what GitHub stored and it seemed that after I uploaded to GitHub the checksum changed from what I generated locally using the /archives link that was in the [spicetify-cli.rb file](https://github.com/khanhas/homebrew-tap/blob/master/spicetify-cli.rb) (https://github.com/spack/spack/issues/5411).

From here, I assumed that there were two possible solutions:
1. Start uploading the SHA256 from what GitHub generates.
2. Change the link to use the /releases endpoint which uses the file generated in the release.

However, I noticed that the filesize also changes, that's when I noticed that that file is the file generated from the tag and that brew, on install, builds the source code instead of using the tar ball generated by me... 

Soooo 🤦 , facepalm moment, I used the wrong SHA256, because what I should have submitted was the checksum from the tag, and not from the release...

**Comment by afonsojramos** [comment]
**TLDR**
https://github.com/khanhas/homebrew-tap/commit/bab47a33649f5ed2d05a08326578f38ff48c13ed fixed #1127

**Comment by afonsojramos** [comment]
@lor1x please do try again and report back 😊

**Comment by lor1x** [comment]
Yes @afonsojramos it worked like a charm, thank you so much for fixing it so fast! Have a great weekend!
